wpiunits/src/main/java/edu/wpi/first/units/Velocity.java

```diff
@@ -7,6 +7,19 @@
 import edu.wpi.first.units.collections.LongToObjectHashMap;
 import java.util.Objects;
 
+/**
+ * Unit of velocity dimension that is a combination of a distance unit (numerator) and a time unit
+ * (denominator).
+ *
+ * <p>This is the base type for units of velocity dimension. It is also used in combination with a
+ * distance dimension to specify the dimension for {@link Measure}. For example:
+ * <code>Measure&lt;Velocity&lt;Distance&gt;&gt;</code>.</p>
+ *
+ * <p>Actual units (such as {@link Units#Watts} and {@link Units#Horsepower}) can be found in the
```

Comment:
`Units#MetersPerSecond` and `Units#RPM` might be more relevant unit examples, power isn't used nearly as much as linear and angular velocity.

Reply:
Oh yeah, that's a copy/paste error. Watts and Horsepower aren't velocities.